LethalAPI.Core/Patches/HarmonyTools/EventTranspilerInjector.cs

```diff
@@ -0,0 +1,145 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="EventTranspilerInjector.cs" company="LethalAPI Modding Community">
+// Copyright (c) LethalAPI Modding Community. All rights reserved.
+// Licensed under the GPL-3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
```

Comment:
If this is taken from Exiled, we should also credit it properly.

Reply:
I made this from scratch.

Reply:
Oh, damn, nice work. Apologies - thought the event system came from Exiled considering the earlier exiled comments / copyright.